### PR TITLE
Sign in and Signup Screens are Added

### PR DESCRIPTION
**Pull request by dev04sa**
In the main project of LibraLink , I created a folder Screens inside the lib folder. Then inside the Screens folder I create another folder Auth in this folder there is Two  files for Login and Signup. 
Preview of screens are given below - 
WhatsApp Image 2023-10-26 at 20 00 56_040a214d

WhatsApp Image 2023-10-26 at 20 00 55_a4c79a2e

https://github.com/HackSlashNITP/LibraLink/assets/129666293/e38c3460-78c5-40c2-b15c-a78d2810e1b1

Please review it and give me feedback.

### CODE REVIEW

File: .vscode/settings.json
Comment (RitwikSingh28) [inline .vscode/settings.json:1]: Looks like we will have to add `.vscode/settings.json` to `.gitignore`, as this is just a dev env setting.

File: lib/Screens/Auth/signup_screen.dart [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+                          const SizedBox(
+                            height: 30,
+                          ),
+                          Column(children: [
+                            SizedBox(
+                              width: 320,
+                              height: 52,
+                              child: TextField(
+                                controller: nameController,
+                                decoration: InputDecoration(
```
Comment (RitwikSingh28) [inline lib/Screens/Auth/signup_screen.dart:87]: The current `highlightColor` of the `TextField` is light bluish, would be better if you can change it to a slightly darker one

**Review by RitwikSingh28 — CHANGES_REQUESTED** [review]
@dev04sa 

Fantastic work! I don't find any major issues with your work, so great going..
However, there are a few minor things that I would like you to change (already outlined in the comments preceding this) before I can accept this PR

**Review by RitwikSingh28 — APPROVED** [review]
Everything correct...
Merging

File: .vscode/settings.json
Comment (dev04sa) [reply .vscode/settings.json:1]: @RitwikSingh28 Sir I am trying the different ways to add the .vscode folder inside the gitignore , But I don't know why I am not able to do so last three commit are regarding to this problem.
sorry for the inconvenience

**Pull request merged**